src/Uno.UWP/Globalization/CurrencyIdentifiers.cs

```diff
@@ -0,0 +1,332 @@
+﻿#nullable enable
+
+namespace Windows.Globalization;
+
+public static partial class CurrencyIdentifiers
+{
+	internal const string HNLCurrencyIdentifier = "HNL";
```

Comment:
Consider inlining these constants.

Reply:
I don't understand. Can you explain what to do?

Reply:
@workgroupengineering Instead of:

```csharp
	internal const string HNLCurrencyIdentifier = "HNL";
	public static string HNL => HNLCurrencyIdentifier;
```

Let's have a single line:

```csharp
	public static string HNL => "HNL";
```